README.md

```diff
@@ -0,0 +1,40 @@
+# Predicting House Prices
+
+This project is about creating an endpoint that predicts house prices based on inputted features. 
+
+
+## Endpoint
+The endpoint `/predict` accepts payload of the following structure.
+``` http
+POST /api/predict/
+```
+|Parameter |  Type         |  Description           |
+|----------|:-------------:|-----------------------:|
+| `inputs` |  `2 D array`  | Features to be predicted of shape (13,2)|
```

Comment:
I suggest you allow the user to call your endpoint using a dict input, because 2D array is very confusing and very error-prone.

``` 
{
    inputs: [
        {
            "MSSubClass": 60,
            "MSZoning":"RM",
            .... and so on

        },
        {
             ....
        },
    ]
}
```